JuiceMaker/JuiceMaker/Controller/ModifyStockViewController.swift

```diff
@@ -2,18 +2,59 @@
 //  ModifyStockViewController.swift
 //  JuiceMaker
 //
-//  Created by Yena on 2023/05/16.
+//  Created by dasan & kyungmin on 2023/05/16.
 //
 
 import UIKit
 
-class ModifyStockViewController: UIViewController {
-
+final class ModifyStockViewController: UIViewController {
+    private var oldFruitStocks: [Int] = [Int]()
+    private var newFruitStocks: [Int] = [Int]()
+    
+    @IBOutlet var fruitStockLabels: [UILabel]!
+    @IBOutlet var fruitStockStepper: [UIStepper]!
+        
     override func viewDidLoad() {
         super.viewDidLoad()
+        updateFruitStockLabel()
+        initializeStepperValue()
+    }
+    
+    @IBAction func touchUpStepper(_ sender: UIStepper) {
+        if let stepperIndex = fruitStockStepper.firstIndex(of: sender) {
+            fruitStockLabels[stepperIndex].text = String(Int(sender.value))
+            newFruitStocks[stepperIndex] = Int(sender.value)
+        }
+    }
+    
+    private func updateFruitStockLabel() {
+        for fruitStockLabel in fruitStockLabels {
+            fruitStockLabel.text = String(newFruitStocks[fruitStockLabel.tag])
+        }
+    }
+    
+    private func initializeStepperValue() {
+        for fruitStockStepper in fruitStockStepper {
+            fruitStockStepper.minimumValue = Double(oldFruitStocks[fruitStockStepper.tag])
+        }
     }
+    
+    private func calculateStockDifference() -> [Int] {
+        var stockDifference: [Int] = [Int]()
+        for (stockIndex, stock) in newFruitStocks.enumerated() {
+            stockDifference.append(stock - oldFruitStocks[stockIndex])
+        }
+        return stockDifference
+    }
+}
 
-    @IBAction func dismissModal(_ sender: UIButton) {
-        dismiss(animated: true, completion: nil)
+extension ModifyStockViewController {
+    func setStock(stocks: [Int]) {
```

Comment:
메서드명에 Stock이 이미 있기 때문에 매개변수로 굳이 또 보여줄 필요는 없어보입니다.
```suggestion
    func setStock(_ stock: [Int]) {
```

Reply:
와일드 카드로 가려줄 생각을 못했네요😅 감사합니다!!